src/legacy/services/profile.service.ts

```diff
@@ -6,7 +6,7 @@ export function fetchOneProfile(profile_id: number) {
     const res = axios.get(BASE_URL + profile_id);
     return res;
   } catch (error) {
-    console.log("hay un error");
+   //
```

Comment:
debe hacer throw del error ya sea del que te manda el backend o de uno personalizado xd, esto nunca fallara si el backend llega a dar error entonces habran errores silenciosos
```suggestion
   throw new BackendError(error);
```